client/elements/page-selector.html

```diff
@@ -87,6 +87,22 @@
         }
       }
 
+      .toast {
+        @apply --sc-skolar-font-size-xl;
+        @apply --sc-sans-font;
+        --paper-toast-color: var(--sc-tertiary-text-color);
+        text-align: center;
+        margin-left: calc(50vw - 150px);
+      }
+
+      .success-toast {
+        --paper-toast-background-color: var(--sc-primary-accent-color);
```

Comment:
`--sc-toast-success-color`. The primary color can change, but the toast should stay green.